ext/js/background/backend.js

```diff
@@ -599,6 +600,24 @@ export class Backend {
         return 'browse';
     }
 
+    /** @type {import('api').ApiHandler<'notesView'>} */
+    async _onApiNotesView({noteIds, mode, allowFallback}) {
```

Comment:
This would probably be simplified if you just replace `noteView` entirely with `notesView`. If you do that, I'd also suggest to probably rename it to `viewNotes`.

Reply:
`noteView` is still used inside `_viewNote` (in `display-anki.js`) for the right-click note selection menu. Switching to `notesView` (or `viewNotes`) would change the call to:

``` js
// pass an array with just the first element
await this._display.application.api.viewNotes(noteIds.slice(0,1), this._noteGuiMode, false);
```

It works but it's not ideal

Reply:
`api.noteView` is also used from `_viewNoteForSelectedEntry` (still in `display-anki.js`). When is this method called?

Clicked a bunch of things but nothing hit my breakpoint.

Reply:
What I mean is just delete the old `noteView` and use the new `viewNotes` in its place. There are only two places in the codebase that it's used, one in display-anki and one in anki-controller.

The first should just be able to be changed to:
```js
await this._display.application.api.viewNotes(noteIds, this._noteGuiMode, false);
```

And the latter to:
```js
await this._settingsController.application.api.viewNotes([noteId], mode, false);
```

Not sure what you mean by "it works but it's not ideal".

> When is this method called?

`_viewNoteForSelectedEntry` is a hotkey action, by default Alt+V. (Seems like the hotkey title text isn't working though.)